src/components/LoadingPlaceholder.vue

```diff
@@ -18,53 +18,56 @@
-			<defs>
-				<linearGradient id="placeholder-gradient">
-					<stop offset="0%" :stop-color="light">
-						<animate attributeName="stop-color"
-							:values="`${light}; ${light}; ${dark}; ${dark}; ${light}`"
-							dur="2s"
-							repeatCount="indefinite" />
-					</stop>
-					<stop offset="100%" :stop-color="dark">
-						<animate attributeName="stop-color"
-							:values="`${dark}; ${light}; ${light}; ${dark}; ${dark}`"
-							dur="2s"
-							repeatCount="indefinite" />
-					</stop>
-				</linearGradient>
-			</defs>
-		</svg>
-
-		<!-- Placeholders -->
-		<li v-for="placeholder in count" :key="placeholder">
-			<svg
-				v-if="type === 'conversations'"
-				class="conversation-placeholder"
-				xmlns="http://www.w3.org/2000/svg"
-				fill="url(#placeholder-gradient)">
-				<circle class="conversation-placeholder-icon" />
-				<rect class="conversation-placeholder-line-one" />
-				<rect class="conversation-placeholder-line-two" :style="{width: `calc(${randWidth()}%)`}" />
-			</svg>
-			<svg
-				v-if="type === 'messages'"
-				class="message-placeholder"
-				xmlns="http://www.w3.org/2000/svg"
-				fill="url(#placeholder-gradient)">
-				<circle class="message-placeholder-icon" />
-				<rect class="message-placeholder-line-one" />
-				<rect class="message-placeholder-line-two" />
-				<rect class="message-placeholder-line-three" />
-				<rect class="message-placeholder-line-four" :style="{width: `calc(${randWidth()}%)`}" />
+		<template v-for="(suffix, gradientIndex) in ['-regular', '-reverse']">
+			<svg :key="'gradient' + suffix" :class="'placeholder-gradient placeholder-gradient' + suffix">
+				<defs>
+					<linearGradient :id="'placeholder-gradient' + suffix">
+						<stop offset="0%" :stop-color="(gradientIndex == 0) ? light : dark" />
+						<stop offset="100%" :stop-color="(gradientIndex == 0) ? dark : light" />
+					</linearGradient>
+				</defs>
 			</svg>
-		</li>
-	</ul>
+
+			<ul :key="'list' + suffix" :class="'placeholder-list placeholder-list' + suffix">
+				<li v-for="(width, index) in placeholderData" :key="'placeholder' + suffix + index">
+					<svg
+						v-if="type === 'conversations'"
+						class="conversation-placeholder"
+						xmlns="http://www.w3.org/2000/svg"
+						:fill="'url(#placeholder-gradient' + suffix + ')'">
+						<circle class="conversation-placeholder-icon" />
+						<rect class="conversation-placeholder-line-one" />
+						<rect class="conversation-placeholder-line-two" :style="{width: `width`}" />
+					</svg>
+					<svg
+						v-if="type === 'messages'"
+						class="message-placeholder"
+						xmlns="http://www.w3.org/2000/svg"
+						:fill="'url(#placeholder-gradient' + suffix + ')'">
+						<circle class="message-placeholder-icon" />
+						<rect class="message-placeholder-line-one" />
+						<rect class="message-placeholder-line-two" />
+						<rect class="message-placeholder-line-three" />
+						<rect class="message-placeholder-line-four" :style="{width: `width`}" />
```

Comment:
The last line doesn't seem to have a random length anymore.

Something like the following makes it work:
```diff
diff --git a/src/components/LoadingPlaceholder.vue b/src/components/LoadingPlaceholder.vue
index 49acbfc0e..e74f28c58 100644
--- a/src/components/LoadingPlaceholder.vue
+++ b/src/components/LoadingPlaceholder.vue
@@ -51,7 +51,7 @@ each other by animating the opacities.
                                                :fill="'url(#placeholder-gradient' + suffix + ')'">
                                                <circle class="conversation-placeholder-icon" />
                                                <rect class="conversation-placeholder-line-one" />
-                                               <rect class="conversation-placeholder-line-two" :style="{width: `width`}" />
+                                               <rect class="conversation-placeholder-line-two" :style="width" />
                                        </svg>
                                        <svg
                                                v-if="type === 'messages'"
@@ -62,7 +62,7 @@ each other by animating the opacities.
                                                <rect class="message-placeholder-line-one" />
                                                <rect class="message-placeholder-line-two" />
                                                <rect class="message-placeholder-line-three" />
-                                               <rect class="message-placeholder-line-four" :style="{width: `width`}" />
+                                               <rect class="message-placeholder-line-four" :style="width" />
                                        </svg>
                                </li>
                        </ul>
@@ -97,7 +97,7 @@ export default {
                        const data = []
                        for (let i = 0; i < this.count; i++) {
                                // generate random widths
-                               data.push(Math.floor(Math.random() * 20) + 30)
+                               data.push('width: ' + (Math.floor(Math.random() * 20) + 30) + '%')
                        }
                        return data
                },
```

Reply:
Nice catch. Despite my compare the old and new this didn't catch my eye.

Tested and pushed.